src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -16,33 +17,13 @@
 
 public class JedisSentinelPool extends JedisPoolAbstract {
 
-  /**
-   * @deprecated This will be private in future.
-   */
-  @Deprecated
-  protected static Logger log = LoggerFactory.getLogger(JedisSentinelPool.class);
+  private static final Logger LOG = LoggerFactory.getLogger(JedisSentinelPool.class);
 
-  @Deprecated protected final GenericObjectPoolConfig<Jedis> poolConfig;
   private final JedisFactory factory;
 
-  @Deprecated protected int connectionTimeout;
-  @Deprecated protected int soTimeout;
-  @Deprecated protected int infiniteSoTimeout;
-
-  @Deprecated protected String user;
-  @Deprecated protected String password;
-  @Deprecated protected int database;
-  @Deprecated protected String clientName;
-
-  @Deprecated protected int sentinelConnectionTimeout;
-  @Deprecated protected int sentinelSoTimeout;
-  @Deprecated protected String sentinelUser;
-  @Deprecated protected String sentinelPassword;
-  @Deprecated protected String sentinelClientName;
-
   private final JedisClientConfig sentinelClientConfig;
 
-  protected final Set<MasterListener> masterListeners = new HashSet<>();
+  protected final Collection<MasterListener> masterListeners = new ArrayList<>();
```

Comment:
Is there any reason to use `Collection` and not `List`?

Reply:
There is no specific reason for it to be `List`. That's why `Collection`.

Reply:
I hope you don't mind. As a Java developer, I see this as a programming habit.

Reply:
XD

Reply:
Me too.
I only shift if it is in only one method and its life is only within that method.

Reply:
I (hopefully) understand what you're indicating. I should've avoided that.

Reply:
That's just a personal opinion. I wouldn't object to your approach. : )